client/blocks/image-editor/image-editor-toolbar.jsx

```diff
@@ -179,21 +208,24 @@ class ImageEditorToolbar extends Component {
 			}
 		];
 
-		return buttons.map( button =>
-			button
+		return buttons.map( button => {
```

Comment:
This was here before, and the usage here is fine, but heads up that we don't polyfill all prototype methods on Array. For other things like `startsWith` we use should a lodash equivalent.